server/sonar-server/src/main/java/org/sonar/server/platform/ServerLogging.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * SonarQube, open source software quality management tool.
+ * Copyright (C) 2008-2014 SonarSource
+ * mailto:contact AT sonarsource DOT com
+ *
+ * SonarQube is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * SonarQube is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonar.server.platform;
+
+import ch.qos.logback.classic.Level;
+import ch.qos.logback.classic.Logger;
+import com.google.common.base.Preconditions;
+import com.google.common.collect.ImmutableSet;
+import java.io.File;
+import java.util.Set;
+import org.slf4j.LoggerFactory;
+import org.sonar.api.config.Settings;
+import org.sonar.api.server.ServerSide;
+import org.sonar.process.LogbackHelper;
+import org.sonar.process.ProcessProperties;
+
+@ServerSide
+public class ServerLogging {
+
+  public static final Set<Level> ALLOWED_ROOT_LOG_LEVELS = ImmutableSet.of(Level.TRACE, Level.DEBUG, Level.INFO);
+
+  private final LogbackHelper helper = new LogbackHelper();
+  private final Settings settings;
+
+  public ServerLogging(Settings settings) {
+    this.settings = settings;
+  }
+
+  public void changeLevel(Level level) {
+    configureLevels(helper, level);
+    LoggerFactory.getLogger(ServerLogging.class).info("Level of logs changed to {}", level);
+  }
+
+  public static void configureLevels(LogbackHelper helper, Level level) {
```

Comment:
given that this method is dependent only on a `LogbackHelper` instance, given a `LogbackHelper` instance is created specifically for calling this method, I really don't think it need to be static.
It could be replaced by a class (called `ServerConfigurationHelper`) wrapping a `LogBackHelper` instance and exposing this `configureLevels` method

(static is evil! "Helper" pattern too but are to help here)